start: reader instructions (in index pages) (#4000)
iterative/dvc.org: content/docs/start/index.md

<details>

### ⚙️ Expand to prepare a project.

Imagine we want to building an ML project from scratch. Let's start by creating
a Git repository:

```dvc
$ mkdir example-get-started
$ cd example-get-started
$ git init
```

<admon type="info">

This name is actually used in our
[example-get-started](https://github.com/iterative/example-get-started) repo.

> </admon>

Author: I also realized this is only applicable to the Data Mgmt trail now so I updated the note here (made it more general) and added hidden tips about each example repo into the trail index pages (context: https://github.com/iterative/dvc.org/issues/2496#issuecomment-1255431925).

<details>

### ⚙️ Expand to prepare a project.

Imagine we want to building an ML project from scratch. Let's start by creating
a Git repository:

```dvc
$ mkdir example-get-started
$ cd example-get-started
$ git init
```

<admon type="info">

> This name is actually used in our

Author: ```suggestion
This directory name is actually used in our
```

Now you're ready to DVC!

The value of DVC's many feature sets is best understood from different angles.
Pick a trail below to see an overview of all features from that perspective:

### Data Management

> - [Introduction](/doc/start/data-management)

Author: Added links to the indices as "introduction".

- **[Data and model versioning]** is the base layer of DVC for large files,
  datasets, and machine learning models. Use a regular Git workflow, but without
  storing large files in the repo (think "Git for data"). Data is stored
  separately, which allows for efficient sharing.

- **[Data and model access]** goes over using data artifacts from outside of the
  project and importing them from another DVC project. This can help to download

>   a specific version of an ML model to a deployment server or import a model to

Author: Updated some of these descriptions to match the cards at the bottom of the trail index pages.

[metrics, parameters, and plots]:
  /doc/start/data-management/metrics-parameters-plots

<admon type="tip">

The steps and results of some of these chapters are captured in our
[example-get-started] repo. Feel free to `git clone/checkout` any of its [tags].

[example-get-started]: https://github.com/iterative/example-get-started
[tags]: https://github.com/iterative/example-get-started/tags

</admon>

### Experiment Management

- **[Experiments]** enable exploration, iteration, and comparison across many
  trials in ML projects. Track your experiments with automatic versioning and
  checkpoint logging. Compare differences in parameters, metrics, code, and
  data. Apply, drop, roll back, resume, or share any experiment.

- **[Visualization]** helps you compare experiment results visually, track your
  plots, and generate them with library integrations.

[experiments]: /doc/start/experiment-management/experiments
[visualization]: /doc/start/experiment-management/visualization

<admon type="tip">

> The are captured in our [example-dvc-experiments] repo (see its [tags]).

Author: ```suggestion
These are captured in our [example-dvc-experiments] repo (see its [tags]).
```

[example-dvc-experiments]: https://github.com/iterative/example-dvc-experiments
[tags]: https://github.com/iterative/example-dvc-experiments/tags

</admon>

## Following the Get Started

> Although presented in a logical order, each page in the trails above is an
> independent chapter you can go to directly. There may be references to other
> chapters or docs, but you can skip them and still get a good idea of the
> features in question, even by only reading page sections.
> 
> For better learning, you can also try each step yourself and get to similar
> results. Some of the required steps for this may be inside collapsed sections

Author: ```suggestion
Each page in the trails above is more or less independent,
especially if you're only reading them to get a general idea of the
features in question. For better learning, try each step yourself from the beginning of any trail.
Some of the preparation steps may be inside collapsed sections
```
